libs/sr/ui/src/LightCone/LightConeCard.tsx

```diff
@@ -14,10 +13,9 @@ type LightConeCardProps = {
 export function LightConeCard({ lightConeId }: LightConeCardProps) {
   const lightCone = useLightCone(lightConeId)
 
-  const member0 = convert(ownTag, { et: 'own', src: '0' })
   const calc = srCalculatorWithEntries(
     withMember('0', ...lightConeData(lightCone))
```

Comment:
We can remove `withMember` and `withTag`, unless you plan to do multiple members here later.

Reply:
Like ?
```Typescript
const calc = srCalculatorWithEntries(lightConeData(lightCone))
```

Reply:
yes